### DISCUSSION TITLE
Using ,mjs file inline

### DISCUSSION THREAD

**Discussion by aimcosoftware** [mozilla/pdf.js#18008] [Q&A]
I am using the .js version of pdf.js, for offline use, which is working.
I'm replacing `{pdf_js.js}` and `{pdf_worker.js}`, with pdf.js and pdf_worker.js, and loading the HTML
``` 
<script id="pdf_js">
	{pdf_js.js}
</script>

<script id="pdf_worker">
	{pdf_worker.js}
</script>
```
That doesn't work with the newer .mjs scripts, using the following doesn't work.
It does work if I load from a server with a src attribute.
```
<script id="pdf_js" type="module">
	{pdf_js.mjs}
</script>

<script id="pdf_worker"  type="module">
	{pdf_worker.mjs}
</script>
```
Can anyone show me what I'm missing.
Thanks

**Comment by TwisterMc** [comment]
You have to add support to your server for mjs files. Try one of the following.

- Add `AddType application/javascript .mjs` to your `.htaccess` file if you’re on an Apache server.
- If you’re on a nginx server, open your `mime-types.conf.hbs` file and update `application/x-javascript js;` to `application/x-javascript js mjs;`
- Contact your hosting support and ask them to add a mime type support for mjs files.
